bangc-ops/mlu_op.h

```diff
@@ -3047,6 +3047,91 @@ mluOpAddN(mluOpHandle_t handle,
           const mluOpTensorDescriptor_t output_desc,
           void *output);
 
+// Group:Pad
+/*!
+ * @brief Pads the input tensor using \b padding_value based on \b paddings that specifies how the tensor is
+ * padded or cropped. This operation is usually used in ResNet and Transformer networks.
+ *
+ * @param[in] handle
+ * Handle to a Cambricon MLUOP context that is used to manage MLU devices and queues in the pad operation. For
+ * detailed information, see ::mluOpHandle_t.
+ * @param[in] input_desc
+ * The descriptor of the input tensor. For detailed information, see ::mluOpTensorDescriptor_t.
+ * @param[in] input
+ * Pointer to the MLU memory that stores the input tensor.
+ * @param[in] paddings
+ * A host pointer to the \b paddings data that holds the padding size to be added for each dimension
+ * of \b input. Positive and negative padding values represent the padding size and the cropping size, respectively.
+ * @param[in] padding_value
+ * A host pointer to the \b padding_value that holds the constant value to be added.
+ * @param[in] output_desc
+ * The descriptor of the output tensor. For detailed information, see ::mluOpTensorDescriptor_t.
+ * @param[out] output
+ * Pointer to the MLU memory that stores the output tensor.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ *
+ * @par Data Type
+ * - Data types of input tensor \b input, \b padding_value and output tensor \b output must be the same. The
+ *  supported data types are as follows:
+ *   - input tensor: uint8, int8, uint16, int16, uint32, int32, uint64, int64, bool, half, float.
+ *   - paddings: int32.
+ *   - padding_value: uint8, int8, uint16, int16, uint32, int32, uint64, int64, bool, half, float.
+ *   - output tensor: uint8, int8, uint16, int16, uint32, int32, uint64, int64, bool, half, float.
+ *
+ * @par Scale Limitation
+ * - Only supports padding with a constant value in \b padding_value.
+ * - The input tensor and output tensor must meet the following requirements:
+ *   - The number of dimensions of \b input is no more than 8.
+ *   - The shape of the integer array of \b paddings is \p [n, 2]. Length of the first dimension of \b paddings is
+ *   \p n, which equals to the number of dimensions of \b input. Length of the second dimension of \b paddings is 2,
+ *   which means the two padding directions of before and after directions for each dimension of \b input. For each
+ *   dimension \p k, \p paddings[k, 0] and \p paddings[k, 1] specify the padding size before and after the input data
+ *   respectively.
+ *   - \p output[k] = \p paddings[k, 0] + \p input[k] + \p paddings[k, 1]. \p k represents each dimension of \b input.
```

Comment:
建议生成文档看下效果，目测这个区域会有问题。。。 可以换成这种格式试试

```suggestion
 *   - ``output[k] =  paddings[k, 0] +  input[k] +  paddings[k, 1]``. \p k represents each dimension of \b input.
```

Reply:
已修改